### PR TITLE
Add presence for Vukkybox (vukkybox.com)

### PR DESCRIPTION
**Pull request by mitchellonii**

### CODE REVIEW

**Comment by EncryptedDev** [comment]
I think you got that metadata file type wrong..

**Comment by EncryptedDev** [comment]
I think you got that metadata file type wrong..

File: websites/V/Vukkybox/presence.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	});
+
+  presence.on("UpdateData", async () => {
+
+	const presenceData: PresenceData = {
+	  largeImageKey: "icon",
+	  smallImageKey: "vukky",
+	  smallImageText: "Playing Vukkybox",
+	  details: "",
+	  state: "",
```
Comment (ImRodry) [inline websites/V/Vukkybox/presence.ts:16]: This is useless
```suggestion
```

File: websites/V/Vukkybox/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
```
Comment (ImRodry) [inline websites/V/Vukkybox/dist/metadata.json:2]:
```suggestion
  "$schema": "https://schemas.premid.app/metadata/1.5",
```

File: websites/V/Vukkybox/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  "service": "Vukkybox",
+  "author": {
+    "name": "Mitchellonii",
+    "id": "490664749315653642"
+  },
+  "category": "games",
+  "logo": "https://vukkybox.com/resources/classicvukkybox.png",
+  "thumbnail": "https://raw.githubusercontent.com/memedmusic/images/main/vukkybox.png",
```
Comment (ImRodry) [inline websites/V/Vukkybox/dist/metadata.json:10]: These must be imgur links

**Comment by memedmusic** [comment]
aaaa why did the rename not work properly

**Review by ImRodry — CHANGES_REQUESTED** [review]
Please install the eslint extension on vscode and fix all the eslint errors, as well as prettier errors

**Comment by memedmusic** [comment]
I believe that should be everything

**Comment by ImRodry** [comment]
Resolve the conversations

File: websites/V/Vukkybox/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.5",
+    "service": "Vukkybox",
+    "author": {
+        "name": "Mitchellonii",
+        "id": "490664749315653642"
+    },
+    "category": "games",
+    "logo": "https://imgur.com/dIpE7wX",
+    "thumbnail": "https://imgur.com/uS5AFMi",
```
Comment (ImRodry) [inline websites/V/Vukkybox/dist/metadata.json:10]: these are not image links
```suggestion
    "logo": "https://i.imgur.com/dIpE7wX.png",
    "thumbnail": "https://i.imgur.com/uS5AFMi.png",
```

**Review by ImRodry — CHANGES_REQUESTED** [review]

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload proof of it working

**Review by Bas950 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**